Makefile

```diff
@@ -1,22 +1,127 @@
-.PHONY: help clean spec mystspec
+# Because env `SHELL` might be 
+# some extremely weird shell
+SHELL = bash
 
-spec:      ## Runs all specs
-	crystal spec
-	crystal run src/myst_cli.cr -- spec/myst/spec.mt
+# Parse makefiles named *.mk in the dir `local`
+# for local extra makefile targets if wanted.
+# It is smart to set MYSTBUILD to your myst dev-build there
+-include local/*.mk
 
-myst-spec: ## Runs just the in-language specs
-	crystal run src/myst_cli.cr -- spec/myst/spec.mt
-	
-build:     ## Builds myst into an executable
-	shards build
+# https://www.gnu.org/software/make/manual/html_node/One-Shell.html
+.ONESHELL:
 
-check:     ## Runs all crystal specs
-	crystal spec/
+# The docs says i should do this
+.SUFFIXES: .cr .mt
```

Comment:
Take a look at https://stackoverflow.com/a/2011058 for why this is useful. A comment like "ensure that only `.cr` and `.mt` files can have suffix rules." would be a bit better, but I honestly don't think it's needed here (we're not converting files between types, so we don't need suffix rules, etc.)

Reply:
I see! So that's what it was for 😅